_posts/0000-01-02-ahneyle007.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "welcome to our second slide
+---
+Your text
```

Comment:
_I am *about to * have an issue raised here_
###First Issue will be in a "table form"
What I know|what I should know
------------.                   --------------
Java| python